app/Controllers/Donation/FindCitiesController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\App\Controllers\Donation;
```

Comment:
Since we want to reuse this end point for memberships, I'd be in favor of putting this in the `Controllers\Validation` namespace/directory